### PR TITLE
[enterprise-4.11] updated versioning for eus updates

### PR DESCRIPTION
**Pull request by obrown1205**
Removed and replaced 4.12 versioning verbiage

Version(s):
4.11

Link to docs preview:
https://docs.openshift.com/container-platform/4.11/updating/preparing-eus-eus-upgrade.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by obrown1205** [comment]
I removed a prereq that suggests:
"Ensure that you are familiar with version-specific prerequisites, such as link:[https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator](https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare%5Badministrator) acknowledgement] that is required prior to updating from {product-title} 4.10 to 4.12."

Page does not exist in 4.11 or 4.10. Suggestions? Can I remove?

File: modules/updating-eus-to-eus-upgrade-cli.adoc [hunk 1]
```diff
@@ -37,18 +37,18 @@ master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
 worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
 ----
 
-. Change to the `eus-4.12` channel by running the following command:
+. Change to the `eus-4.10` channel by running the following command:
```
Comment (shellyyang1989) [inline modules/updating-eus-to-eus-upgrade-cli.adoc:40]: Hmm, per [slack discussion](https://redhat-internal.slack.com/archives/CJ1J9C3V4/p1678810330149369), seems like we want to document EUS upgrade from 4.10 to 4.12 in 4.11 doc. cc @evakhoni to take a look.

Comment (obrown1205) [reply modules/updating-eus-to-eus-upgrade-cli.adoc:40]: Sorry for the confusion - will update in Slack :)

**Review by adellape — APPROVED** [review]

**Pull request merged**